### PR TITLE
Add test which uses [tags] in SQL expressions

### PR DESCRIPTION
**Pull request by PhilippGrashoff**
This adds a test which uses [tags] in SQL expressions.

### CODE REVIEW

**Comment by georgehristov** [comment]
Code should be following I believe (provided Sql persistence, no changes in latest versions):

```
$model->addExpression(
            'full_name_with_salutation',
            $model->expr(
                "CONCAT([salutation], ' ', [full_name])"
            )
        );
```

**Comment by mvorisek** [comment]
@PhilippGrashoff very important - never, never, change CS in specific PRs. If you belive we need to change some CS, propose as a separate PR.

File: tests/ExpressionSqlTest.php [hunk 1]
```diff
@@ -239,4 +252,43 @@ public function testExpressionActionAlias()
         $q = $m->action('fx0', ['sum', 'x']);
         $this->assertEquals([0 => ['sum_x' => 5]], $q->getRows());
     }
+
+    public function testTagsInBracketsCanBeUsedInLiteralSql()
```
Comment (mvorisek) [inline tests/ExpressionSqlTest.php:256]: name implies https://github.com/atk4/dsql/pull/264 which I think is unrelated

**Comment by mvorisek** [comment]
please describe in desc what expr are input and what expr should be output - it seems you ask for some dsql feature

**Comment by PhilippGrashoff** [comment]
Thought there was some issue in atk data, but basically lagged the knowledge that Model::expr() is a dynamic method added by Persistence\SQL

**Comment by mvorisek** [comment]
> 
> 
> Thought there was some issue in atk data, but basically lagged the knowledge that Model::expr() is a dynamic method added by Persistence\SQL

does it makes any difference? I am still missing expected input -> output.

**Comment by PhilippGrashoff** [comment]
I basically used tags as in test in my model and they failed, but this was due to not using Model::expr(). I don't see this test as too useful any more as it mainly should test if `[tagsinbrackets]` are used correctly everywhere. Having this additional test I wrote in some class with other non-related Tests does not seem to good any more.

**Comment by mvorisek** [comment]
I mean added expr `first_plus_second` should be usable in another expr. If that is NOT the case, I do not understand the issue.

**Comment by PhilippGrashoff** [comment]
there is no issue, as this works. I thought it wouldnt, but that was due to the fact that I thought Model::expr() was removed and used ->persistence->connection->expr().

**Comment by mvorisek** [comment]
I see, for record:
image

**Pull request closed without merging**